bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.2.0",
-  "framework": ">=2.0.15",
+  "version": "2.2.1",
+  "framework": ">=3",
```

Comment:
Sorry may well be missing something obvious but what is it here that requires FW v3+?